projects/demo/project.json

```diff
@@ -80,8 +80,9 @@
                     "echo 'Github pages require special baseHref + 404.html'",
                     "echo 'Read more: https://angular.io/guide/deployment#deploy-to-github-pages'",
                     "echo ------",
-                    "nx build --base-href='/maskito/'",
```

Comment:
cli flag `--base-href='/maskito/'` does not properly work for prerender commands